src/stores/widgets/SpectralProfileWidget/SpectralProfileWidgetStore.ts

```diff
@@ -334,6 +334,15 @@ export class SpectralProfileWidgetStore extends RegionWidgetStore {
             }
         );
 
+        reaction(
```

Comment:
I realized in the moment generator that when we change the coordinate system, the values on the textbox do not update automatically so I added this fix.